src/main/app.ts

```diff
@@ -28,17 +28,15 @@ new Nunjucks(developmentMode)
 // secure the application by adding various HTTP headers to its responses
 new Helmet(config.get("security")).enableFor(app);
 
-// view engine setup
-app.set("views", path.join(__dirname, "views"));
-
-app.use(express.static(path.join(__dirname, "public")));
 app.use(favicon(path.join(__dirname, "/public/img/favicon.ico")));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: false}));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, "public")));
-
-
+app.use((req, res, next) => {
+    res.setHeader('Cache-Control', 'no-cache, max-age=0, must-revalidate, no-store');
```

Comment:
this seems a bit nuclear? don't cache anything?

Reply:
it is still not applicable to static files. Isn't that fine?

Reply:
ah ok, its fine if this middleware doesn't affect static files